integration/typings_test_ts41/include-all.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+
+
+import * as animations from '@angular/animations';
+import * as animationsBrowser from '@angular/animations/browser';
+import * as animationsBrowserTesting from '@angular/animations/browser/testing';
+import * as common from '@angular/common';
+import * as commonHttp from '@angular/common/http';
+import * as commonTesting from '@angular/common/testing';
+import * as commonHttpTesting from '@angular/common/testing';
+import * as compiler from '@angular/compiler';
+import * as compilerTesting from '@angular/compiler/testing';
+import * as core from '@angular/core';
```

Comment:
What about `@angular/core/global` 😇:

```suggestion
import * as core from '@angular/core';
import * as coreGlobal from '@angular/core/global';
```

(And also export below.)

Reply:
Thinking about this some more, I think for the purposes of this PR (enabling TS 4.1 use) we should keep the 4.0 and 4.1 tests consistent.
And then a follow-up PR can fix both (potentially in a more long-term maintainable fashion).  Components, for instance, has a more generic approach to the concept: https://github.com/angular/components/tree/bb2bb9c3065ec1a7a845de072305893a7378383a/integration/ts-compat